conf/app.routes

```diff
@@ -69,4 +69,5 @@ GET        /deployments/:env                            uk.gov.hmrc.cataloguefro
 GET        /health-indicators                           uk.gov.hmrc.cataloguefrontend.healthindicators.HealthIndicatorsController.indicatorsForRepoType()
 GET        /health-indicators/:name                     uk.gov.hmrc.cataloguefrontend.healthindicators.HealthIndicatorsController.breakdownForRepo(name: String)
 
-GET        /platform-initiatives                        uk.gov.hmrc.cataloguefrontend.platforminitiatives.PlatformInitiativesController.platformInitiatives(display: DisplayType ?= DisplayType.Chart)
+GET        /platform-initiatives                        uk.gov.hmrc.cataloguefrontend.platforminitiatives.PlatformInitiativesController.platformInitiatives(team = "", display: DisplayType ?= DisplayType.Chart)
+GET        /platform-initiatives/:team                  uk.gov.hmrc.cataloguefrontend.platforminitiatives.PlatformInitiativesController.platformInitiatives(team: String, display: DisplayType ?= DisplayType.Chart)
```

Comment:
Do we need two routes? A single one with an optional query-param should make it simpler.

Reply:
Whittled down to one route